### PR TITLE
Story #15: As a User, I can view tweets I posted on my profile

### PR DESCRIPTION
**Pull request by shrmnk**
## What
As a User, I can view tweets I posted on my profile

## Why
image

---

**Before submitting, check that:**

 - [X] You have written good* commit messages.
 - [X] You have squashed relevant commits together.
 - [X] You have ensured that RuboCop is passing.
 - [X] Your PR relates to one subject, with a clear title and
       description using complete, grammatically correct sentences.
 - [X] You have included a screenshot of the new view.

### CODE REVIEW

File: app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -2,6 +2,17 @@ class TweetsController < ApplicationController
 
   before_action :authenticate_user!, only: [:new, :create]
 
+  def index
+    @user = User.find(params[:user_id])
+    # Order the tweets in reverse-chronological order
```
Comment (tbssc) [inline app/controllers/tweets_controller.rb]: remove comment -- the code is clear enough to tell any developer that it is indeed ordered with latest tweet first

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+
+  def show
+    @user = User.find(params[:id])
+    # Order the latest 5 tweets in reverse-chronological order
```
Comment (tbssc) [inline app/controllers/users_controller.rb]: remove comment

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    @tweets = @user.tweets.last(5).reverse
```
Comment (tbssc) [inline app/controllers/users_controller.rb]: can be written as `@user.tweets.order(created_at: :desc).limit(5)`. This orders the tweets with latest first. And limits the result to 5.  We do the ordering on the database level. don't need to use ruby to reverse the result's order

File: app/views/tweets/_tweet.html.slim [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+                img.media-object.img-thumbnail src="https://placeimg.com/50/50/people" alt="Profile Picture"
+            .media-body
+              h4.media-heading = @user.email
+              | @
+              = @user.email
+        .col-xs-2.text-right
+          = link_to "Profile", user_path(@user), class: 'btn btn-info'
+      .row
+        .col-xs-offset-1.col-xs-10
+          h4 = tweet.body
```
Comment (tbssc) [inline app/views/tweets/_tweet.html.slim]: think a better semantic tag is `p`, then you can bold it in css

**Comment by shrmnk** [comment]
Addressed the code review issues, updated PR screenshot with new view

**Comment by shrmnk** [comment]
LGTM? 😄

**Pull request merged**